sdk/identity/azure-identity/tests/test_live.py

```diff
@@ -24,26 +23,24 @@ def get_token(credential):
     assert token.expires_on
 
 
-def test_certificate_credential(live_certificate):
-    tenant_id = live_certificate["tenant_id"]
-    client_id = live_certificate["client_id"]
+@pytest.mark.parametrize("certificate_fixture", ("live_pem_certificate", "live_pfx_certificate"))
+def test_certificate_credential(certificate_fixture, request):
```

Comment:
What's `request` here?

Reply:
One of pytest's built-in fixtures: https://docs.pytest.org/en/6.2.x/reference.html?highlight=fixture#request